README.md

```diff
@@ -72,10 +72,19 @@ Add the library to dependencies:
 <dependency>
     <groupId>org.jetbrains.kotlinx</groupId>
     <artifactId>kotlinx-io-core-jvm</artifactId>
-    <version>0.5.4</version>
+    <version>0.6.0</version>
 </dependency>
 ```
 
+### JPMS support
```

Comment:
Is it worth explicitly mentioning it? @qwwdfsad

Reply:
I don't have an opinion on that